tests/test_resources/test_modules/test_functions/test_aws_lambda/test_aws_lambda.py

```diff
@@ -0,0 +1,554 @@
+import logging
+import os
+import unittest
+from pathlib import Path
+
+import boto3
+import pytest
+
+import runhouse as rh
+
+logger = logging.getLogger(__name__)
+CUR_WORK_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_RESOURCES = f"{CUR_WORK_DIR}/test_helpers/lambda_tests"
+
+CRED_PATH = f"{Path.home()}/.aws/credentials"
+DEFAULT_REGION = "us-east-1"
+
+if Path(CRED_PATH).is_file():
+    LAMBDA_CLIENT = boto3.client("lambda")
+else:
+    LAMBDA_CLIENT = boto3.client("lambda", region_name=DEFAULT_REGION)
+IAM_CLIENT = boto3.client("iam")
+LAMBDAS_NAMES = set()
+
+
+def test_create_and_run_no_layers():
```

Comment:
I needed to add `env=["httpx"]` to prevent a`ModuleNotFoundError`

Reply:
Doesn't reproduce on my local machine, maybe should check if it reproduces for others.